fenix.json.example

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
how about `config-fenix.json` then to have all different config files with the same prefix, so they can be found easily and are grouped together in a directory listing?